backends/bmv2/jsonconverter.cpp

```diff
@@ -2256,6 +2277,26 @@ Util::IJson* JsonConverter::convertParserStatement(const IR::StatOrDecl* stat) {
                     return result;
                 }
             }
+        } else if (minst->is<P4::ExternFunction>()) {
+            auto extfn = minst->to<P4::ExternFunction>();
+            if (extfn->method->name.name == IR::ParserState::verify) {
+                result->emplace("op", "verify");
+                BUG_CHECK(mce->arguments->size() == 2, "%1%: Expected 2 arguments", mce);
+                {
+                    auto cond = mce->arguments->at(0);
+                    // false means don't wrap in an outer expression object, which is not needed
+                    // here
+                    auto jexpr = conv->convert(cond, true, false);
+                    params->append(jexpr);
+                }
+                {
+                    auto error = mce->arguments->at(1);
+                    BUG_CHECK(error->is<IR::Member>(), "Error when processing verify");
```

Comment:
Maybe this should be an ::error instead of a bug. One can declare variables with an error type, and one could use here an expression with an error result. The ::error message should indicate that only constant error values are supported in verify.